### ISSUE TITLE
Add Tripwire system

### ISSUE THREAD

**Issue by jvanbaarsen** [intercity/intercity-next#221]
In order to improve security, we want to monitor the state of our servers. This includes adding a tripwire system that keeps monitoring certain files to see if they get modified. A possible system for this can be Tripwire.

**Comment by michiels** [comment]
I think that a tool like "rkhunter" includes both service/port scanning, file scanning and known exploit scanning. So with running something like rkhunter, you have all tools in one.

**Comment by jvanbaarsen** [comment]
I believe there is a subtle difference in rkhunter and Tripwire. Tripwire acts as an Intrusion detection system, RKHunter mainly checks for rootkits. I have to dive deeper into this matter. I'll report back here when I have more info.

**Comment by jvanbaarsen** [comment]
@michiels I think it makes sense to run both.

**Comment by michiels** [comment]
@jvanbaarsen Allright. I've already done some research towards this, so if you have anything that you think Tripwire does on top of RKHunter, then I'm happy to hear!

**Comment by jvanbaarsen** [comment]
@michiels Tripwire checks a full set of files and checks if they have been modified. What I've seen so far is that RKHunter checks against a known list of rootkits, and only checks for those instances.

**Comment by jvanbaarsen** [comment]
Downside of Tripwire is that it takes a lot of configuration work.

**Comment by michiels** [comment]
@jvanbaarsen Ok, my experience is that rkhunter checks for all system files that have been installed when you "snapshotted" your system, and you can exclude paths that you know are going to change (because of deployments for example). But, we can test them both as part of this issue!

**Comment by jvanbaarsen** [comment]
@michiels Yeah, I have now configured Tripwire on a new server, will also install RKHunter besides it. We can then see which gives the best result.
